Readme.md

```diff
@@ -14,7 +14,7 @@ catalog, building towards a web page for every book ever published.
      - [The Backend](#the-backend)
      - [The Service Architecture](#the-service-architecture)
    - [Developer's Guide](#developers-guide)
-   - [Running Tests](#running-tests)
+   - [Running Tests](https://github.com/internetarchive/openlibrary/wiki/Testing)
    - [Contributing](#contributing)
    - [Public APIs](https://openlibrary.org/developers/api)
    - [FAQs](https://openlibrary.org/help/faq)
```

Comment:
Can we move FAQs as well to the Wiki? It'll be much better over there!

Reply:
sounds good to me

Reply:
The FAQ is part of the live site, I don't think a dev wiki is an appropriate location. If the intent is to move out of date content out of the repo, perhaps have a stripped down placeholder to indicate the live site expects a page at that location, and provide a link to the live page to aid developers, but make it clear the FAQ content is stored in the production db.